torch/jit/_script.py

```diff
@@ -53,6 +53,57 @@
     def Attribute(value, type):  # type: ignore
         return value
 
+Attribute.__doc__ = """
+    This method is a pass-through function that returns `value`. It is mostly
+    used to indicate to the TorchScript compiler that the left-hand side
+    expression is a class attribute with type of `type`.
+
+    Though TorchScript can infer correct type for most Python expressions, there are some cases where
+    type infernece can be wrong, including:
+    - Empty containers like `[]` and `{}`, which TorchScript assumes to be container of `Tensor`s
+    - Optional types like `Optional[T]` but assigned a valid value of type `T`, TorchScript would assume
+        it is type `T` rather than `Optional[T]`
+
+    In eager mode, it is simply a pass-through function that returns `value`
+    without other implications.
+
+    Example:
+
+    .. testcode::
+
+        import torch
+        from typing import Dict
+
+        class AttributeModule(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.foo = torch.jit.Attribute(0.1, float)
```

Comment:
You used `d = torch.jit.annotate(Dict[str, int], {})` in the previous example, should the type go first or go second? Please make the examples consistent

Reply:
Yeah, the ordering are reversed between `jit.annotate` and `jit.Attribute`. I think it is too late to make them consistent .......

unless we want to take some (calculated) risks to make one of them tolerate both ordering by checking which argument is from `typing` module and intelligently use it as `the_type`

Thoughts?

Reply:
Let's leave them as they are.